server/src/game/end_game_condition.rs

```diff
@@ -2,26 +2,113 @@ use super::{role_list::Faction, role::Role};
 
 #[derive(Debug, PartialEq, Eq, Clone, Hash)]
 pub enum EndGameCondition {
-    Faction(Faction),
-    Role(Role),
+    Town,
+    Mafia,
+    Cult,
 
-    None
+    Arsonist,
+    Werewolf,
+    Ojo,
+
+    Death,
+    Politician,
+
+    Draw
 }
 impl EndGameCondition {
-    pub fn can_win_together(a: EndGameCondition, b: EndGameCondition)->bool{
-        a == Self::None || b == Self::None || a == b
+    pub fn all()->Vec<EndGameCondition>{
+        vec![
+            EndGameCondition::Town,
+            EndGameCondition::Mafia,
+            EndGameCondition::Cult,
+            EndGameCondition::Arsonist,
+            EndGameCondition::Werewolf,
+            EndGameCondition::Ojo,
+            EndGameCondition::Death,
+            EndGameCondition::Politician
+        ]
+    }
+    ///either return Some(EndGameCondition) or None (if the game is not over yet)
```

Comment:
Why?

Reply:
If you are responding to the comment
I will use this later to send a message saying who won and stuff